src/pset/mod.rs

```diff
@@ -91,13 +91,47 @@ impl PartiallySignedTransaction {
         self.global.tx_data.input_count += 1;
         self.inputs.push(inp);
     }
+    /// Read accessor to inputs
+    pub fn inputs(&self) -> &[Input] {
+        &self.inputs
+    }
+    /// Mutable accessor to inputs
+    pub fn inputs_mut(&mut self) -> &mut [Input] {
+        &mut self.inputs
+    }
+    /// Remove the input at `index` and return it if any, otherwise returns None
+    /// This also updates the pset global input count
+    pub fn remove_input(&mut self, index:usize) -> Option<Input> {
```

Comment:
Could you put newlines between the methods and a space after this colon? And the similar colon for outputs :)

Reply:
updated